community/command-line/src/main/java/org/neo4j/commandline/admin/AdminTool.java

```diff
@@ -104,9 +104,9 @@ public void execute( Path homeDir, Path configDir, String... args )
 
     private void badUsage( AdminCommand.Provider command, IncorrectUsage e )
     {
+        failure( e.getMessage() );
```

Comment:
This is not going to work -- `failure()` calls `System.exit()`.

There is also another version of `badUsage()` where the order needs to change.